src/base/logging/log.jl

```diff
@@ -1,6 +1,6 @@
 
 """
-    macro _make_logging_group(sym::Symbol, doc::String)
+$(TYPEDSIGNATURES)
```

Comment:
Q: is there any way to use TYPEDSIGNATURES in the macro that actually generates the functions (below in this file)?